.github/workflows/plugins-test.yaml

```diff
@@ -118,6 +118,8 @@ jobs:
         run: bash test/plugin/run.sh netty-socketio-scenario
       - name: Run rabbitmq-scenario (12)
         run: bash test/plugin/run.sh rabbitmq-scenario
+      - name: Run gson-scenario (6)
```

Comment:
If 2.8.6 supported, this should be changed too.

Reply:
Got it .